.github/workflows/test_release_publish.yml

```diff
@@ -0,0 +1,50 @@
+name: Build, Test, Publish Github and PyPI Releases
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish_github_release_and_pypi:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out the repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Install pip and pipenv
+      run: |
+        python -m pip install --root-user-action=ignore --upgrade pip
+        pip install --root-user-action=ignore pipenv
+        make venv
+
+    - name: Build Release tar.gz
+      run: |
+        pipenv run python setup.py sdist
+
+    - name: Install Build and Run PAT Tests
+      run: |
+        pipenv run pip install --root-user-action=ignore dist/panther_analysis_tool-*.tar.gz
```

Comment:
I think this is safe because we always blow away the working dir before starting

Reply:
+1 the dist directory is always empty on a fresh GHA run (fresh docker container, etc..)